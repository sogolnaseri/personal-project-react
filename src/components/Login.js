import React from "react";
import TextField from "./TextField.js";
import Button from "./Button.js";

export default ({ handleChange, handleLogin, username }) => {
  return (
    <div className="login">
      <h2>Github UserName :</h2>
      <TextField
        name="username"
        handleChange={handleChange}
        id="github-username"
        value={username}
      />
      <div className="Get User button">
        <Button value="GET USER" handleClick={handleLogin} />
      </div>
    </div>
  );
};
