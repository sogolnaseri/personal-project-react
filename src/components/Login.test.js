import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Login from "./Login";

describe("login", () => {
  test("should render properly", () => {
    const renderer = new ShallowRenderer();
    const element = renderer.render(<Login />);
    expect(element).toMatchSnapshot();
  });
});
