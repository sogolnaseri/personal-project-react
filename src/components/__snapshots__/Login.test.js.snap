// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login should render properly 1`] = `
<div
  className="login"
>
  <h2>
    Github UserName :
  </h2>
  <_default
    id="github-username"
    name="username"
  />
  <div
    className="Get User button"
  >
    <_default
      value="GET USER"
    />
  </div>
</div>
`;
